### PR TITLE
fix: no focus after display horizontal paging page content

### PR DESCRIPTION
**Pull request by pspaczek**
New focus functionality in this PR. 
* When you use menu item and move to the "Item" content focus will be set before "Item" content.  
* When you back from "Item" to menu, menu item will be focused.

### CODE REVIEW

File: src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue [hunk 1]
```diff
@@ -226,6 +226,7 @@ const menuItems = computed<MenuItemAttrsProps[]>(() => itemsAsArray.value.map((i
         item,
       ];
     },
+    tabindex: isActive.value ? '-1' : null,
```
Comment (pspaczek) [inline src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue:229]: @AlanLes I have one TS error; Can you help me and know how to fix it?

Comment (AlanLes) [reply src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue:229]: Hello, sure what's the error? :)

Comment (AlanLes) [reply src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue:229]: I'm afraid I can't fetch your remote locally on my machine; I have no access rights and can't investigate it ;/ 
Maybe we'll huddle later on and investigate it together? :)

Comment (AlanLes) [inline src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue:229]: It does the job :) 
```suggestion
    tabindex: isActive.value ? '-1' : undefined,
```

**Pull request merged**